### ISSUE TITLE
Active playlist /now playing  won't  open 

### ISSUE THREAD

**Issue by Jesterof84** [Malopieds/InnerTune#32]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/z-huang/InnerTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

Start any music , attempt to open now playing 

### Expected behavior

Active playing list opens

### Actual behavior

 no response  
If I lock and unlock device directly after command  now playing opens eventually  , may take repeated attempts

### Screenshots/Screen recordings

https://github.com/Malopieds/InnerTune/assets/165532540/e51e999d-15c3-4401-8773-0dcc4925c655

### Logs

Im not quite talented enough to pull logs ..yet

### InnerTune version

5.5

### Android version

Android 12

### Additional information

new Sonim XP10   , Nova launcher , 
And so far this bug is specific to this device both with ver 5.5 or 5.3  my Cat s62 will function normal

**Comment by Malopieds** [comment]
I cannot reproduce it. I will need some logs to know exactly what happened.
To do so, you can install [android debug bridge (adb)](https://developer.android.com/tools/adb) depending on your machine. Enable it on you device. Once everything is done you can type this command : `adb logcat -d | grep 'com.zionhuang.music' -B 100 -A 100 > log` it will produce a file called log. This command is for linux, I don't know for windows. If you can't find any information I think `adb logcat -d > log` should work on windows. Send it here for further examination.

**Comment by Jesterof84** [comment]
[logcat.log](https://github.com/Malopieds/InnerTune/files/14817266/logcat.log)

I tried, hopefully its helpful

**Comment by Malopieds** [comment]
As I understand the logs, you tried to uninstall the app  and install again ? Which version have you tried ? It seems like you tried the original innertune, because there are trace of FireBase, dependency that is not included in my version.

**Comment by Jesterof84** [comment]
I have tried both your version and original and bothe are affected , i wasn't sure how long to run the log for and im in windows adb i couldn't figure how to just log the app .currently installed is yours  ,with or without restoring backup it has this bug

**Comment by Jesterof84** [comment]
[logcat.log](https://github.com/Malopieds/InnerTune/files/14827372/logcat.log)

This for sure is your version

**Comment by Malopieds** [comment]
That's odd, there is no information in the logs about this. I will try later this week to produce an apk that allows you to debug it easier, I will need you to spam-click the queue button. Other buttons works right (like the lyrics for example, or the 3 dots)? Also, if you click in the lower area, but not on the button, does it triggers anything ?

**Comment by Jesterof84** [comment]
From the buttons i used regularly that is the only one That's buggy ,

**Comment by Jesterof84** [comment]
Well .. It fixed it self for now.. kinda.. , I cant see how this is even relevant. My memory card was formatted as internal storage and evidently 3 phones and 5 years was the end for it It corrupted this morning had be ejected and removed now the app works normally , this makes absolutely no sense logically with deleting and reinstalling the app quite a few times and having the same bug. I don't know if it will stay functioning  when I get a new card  hopefully  it does

**Comment by Malopieds** [comment]
This is really weird, I've never seen those kind of repercussion with a corrupted/old sd card...

Closing it for now. If you get again this error feel free to open it again.
